Asterix/wfsc/correction_loop.py

```diff
@@ -289,8 +289,9 @@ def correction_loop_1matrix(testbed: Testbed,
 
             if mode > corrector.total_number_modes:
                 if not Search_best_Mode:
-                    print("You cannot use a cutoff mode ({:d}) larger than the total size of basis ({:d})".
-                          format(mode, corrector.Gmatrix.shape[1]))
+                    print(
+                        f"You cannot use a cutoff mode ({mode:d}) larger than the total size of basis ({corrector.Gmatrix.shape[1]:d})"
+                    )
```

Comment:
This is really, really ugly... can we ignore the linter here? Or at least align the closing parenthesis with the opening parenthesis.

Reply:
i think in yapf, there's a bit of leaway to avoid weird cases. I'll try to make it better but if the autocorrect force it like that, I don't want to correct it back each time I apply it

Reply:
apparently you're out of luck the auto-formating likes it that way ! I can cut the string in 2 parts if you prefer

Reply:
Nah it's fine, don't bother.